docs/src/docs/demos/core/Hr/hr.demos.tsx

```diff
@@ -76,3 +76,94 @@ export function HrSizesDemo() {
     </CodeDemo>
   );
 }
+
+const subHeaderCode = `import React from 'react';
+import { Hr } from '@mantine/core';
+function Demo() {
+  return (
+    <>
+      <Hr subHeader="Test" />
+      <Hr
+        subHeader="Test"
+        subHeaderProps={{ style: { textAlign: 'center' }, color: 'red' }}
+        style={{ marginTop: 10 }}
```

Comment:
Do not include margins in examples code, it does not add value, just brings a noise